### DISCUSSION TITLE
Opportunistic Snoozing?

### DISCUSSION THREAD

**Discussion by jdewolfe** [NightscoutFoundation/xDrip#3031] [General]
Last 2 nights I've had alerts not fire due to "Opportunistic Snoozing" after working during the day and am not sure what is causing this.

**Comment by Navid200** [comment]
How have you concluded that this is due to opportunistic snoozing?

**Comment by jdewolfe** [comment]
My conclusion is based on inspecting the logs after waking up at 2:25 with a BG of 48 when the alarm is set to 70



![image](https://github.com/NightscoutFoundation/xDrip/assets/28075111/ef407d76-8876-4594-bb1a-dfcc7abb27dd)

**Reply by Navid200** [reply]
Can I see a screenshot of your Glucose level alert list page?

**Comment by jdewolfe** [comment]
This one?

![image](https://github.com/NightscoutFoundation/xDrip/assets/28075111/4e6c618a-2eae-43e6-ad65-63b398ef34d1)

**Reply by Navid200** [reply]
70 and 67?!

Why?  
What's the point of having two low alerts so close together?  What is your intent?

**Comment by jdewolfe** [comment]
Originally the alarm was 67 but that seemed like too low of a warning so 70 was added with a different alarm to test whether it would be louder and wake me up quicker.  I left it as the first alarm generally fires if the number is dropping slowly into that range and the second lower alarm will fire if the number keeps dropping lower or is dropping more aggressively beyond the alarm.   Overall redundant so can remove if it helps the software function better.

**Reply by Navid200** [reply]
What is your hardware data source?

**Comment by jdewolfe** [comment]
Overall my setup uses anubis g6 transmitter, orangelink, omnipod eros, and pixel 6a phone

**Reply by Navid200** [reply]
Do you have BlueJay?

**Comment by jdewolfe** [comment]
Yes

**Reply by Navid200** [reply]
Are you wearing it at night in bed?
If you snooze an alert by pressing on the BlueJay button, xDrip will show the opportunistic snooze log.

It is possible that you are snoozing the alert without remembering.
We may have had this conversation before.  Here is my suggestion.

Set up a sound recording device where you sleep.  If/when this happens again where you see logs of snoozing alerts, play the sound recorder and see if you can hear the alert.

**Comment by jdewolfe** [comment]
I'll run the test and check whether the prior discussion also had an
opportunitistic snooze
